client/src/routes/Session/components/HostNotes/HostNotes.tsx

```diff
@@ -127,19 +153,26 @@ const HostNotes: React.FC<HostNotesProps> = ({
           <TopSafeArea />
           <Gutters>
             <Spacer4 />
-            <TopBar>
-              <Progress
-                index={sessionSlideState?.index}
-                length={exercise?.slides.length}
-              />
-              <Spacer8 />
-              <ToggleButton
-                disabled={!notes}
-                isToggled={showNotes}
-                title={t('notes')}
-                onPress={() => setShowNotes(prevShowNotes => !prevShowNotes)}
-              />
-            </TopBar>
+            <GestureRecognizer onSwipeDown={() => setShowNotes(true)}>
+              <TopBar>
+                <Progress
+                  index={sessionSlideState?.index}
+                  length={exercise?.slides.length}
+                />
+                <Spacer8 />
+                <ToggleButton
+                  disabled={!notes}
+                  isToggled={showNotes}
+                  title={t('notes')}
+                  onPress={() => setShowNotes(prevShowNotes => !prevShowNotes)}
+                />
+              </TopBar>
+              {!showNotes && notes && (
+                <HandleContainerTopBar>
+                  <Handle />
+                </HandleContainerTopBar>
+              )}
+            </GestureRecognizer>
```

Comment:
It feels a little weird to only react on a gesture when there's a handle/grabber. But let's start like this :)

Reply:
Agree, but it's simple :)